src/olympia/editors/views.py

```diff
@@ -645,9 +640,15 @@ def version(self):
     num_pages = pager.paginator.num_pages
     count = pager.paginator.count
 
-    try:
-        flags = ViewQueue.objects.get(id=addon.id).flags
-    except ViewQueue.DoesNotExist:
+    if version:
+        version.admin_review = addon.admin_review
+        version.requires_restart = any(
+            file_.requires_restart for file_ in version.all_files)
+        version.sources_provided = bool(version.source)
+        version.is_webextension = any(
+            file_.is_webextension for file_ in version.all_files)
+        flags = get_flags(version)
```

Comment:
I almost added them, but that would have meant even more tests to write, more refactoring...